hw_3.Functions.py

```diff
@@ -0,0 +1,158 @@
+# 1) Write a function that emulates the game "rock, scissors, paper"
+# At the entrance, your function accepts your version printed from the console, the computer makes a decision randomly.
+
+def game_rockscissorspaper():
+    import random
```

Comment:
It is not an error to import the module inside the function if you are using it in this function only. However, most Python style guides encourage programmers to place all imports at the beginning of the module file.